### DISCUSSION TITLE
search console에 색인생성 잘 되시나요?

### DISCUSSION THREAD

**Discussion by thebbong** [hyesungoh/comet-land#112] [Q&A] [answered]
발견됨 - 현재 색인이 생성되지 않음
상태: 제외됨
최종크롤링은 해당사항없음으로 나오는데...
수동으로 색인생성을 요청하면 잘 되긴 합니다. 그런데, 글 작성할때마다 그럴수도 없고...
계속 뒀는데, 한달넘게 이러네요 😂
sitemap은 제출한 상태에요.

**Comment by hyesungoh** [comment, marked as answer]
저는 색인이 생성되고 있긴 합니다. 

> 제 경우에는 전에 사용하던 블로그 형식을 같은 도메인에서 유지하고 있어서, path 변경된 부분이 많아 조금 걸리고 있긴 합니다.

조금 찾아보니, '발견됨 - 현재 색인이 생성되지 않음'의 경우 취할 수 있는 방법으로

기다리거나, 페이지별로 색인 요청을 하는 것이라고 하네요. 

[참고 링크](https://sy1s.tistory.com/entry/%EA%B5%AC%EA%B8%80-%EC%84%9C%EC%B9%98-%EC%BD%98%EC%86%94-%EC%83%89%EC%9D%B8-%EC%83%9D%EC%84%B1%EC%A4%91-%EB%B0%9C%EA%B2%AC%EB%90%A8-%ED%98%84%EC%9E%AC-%EC%83%89%EC%9D%B8%EC%9D%B4-%EC%83%9D%EC%84%B1%EB%90%98%EC%A7%80-%EC%95%8A%EC%9D%8C-%ED%95%B4%EA%B2%B0%EB%B2%95)

---

저는 해당 상태의 색인이 나오지 않으며, 페이지별로 신청하지 않은 색인도 생성이 되는 것으로 보아...

서치 콘솔이 항상 그렇듯, 조금 기다려보시면 해결되지 않을까 조심스럽게 예상해봅니다.

> 앞으로 색인에 대해 서치 콘솔의 소규모 업데이트가 있을 예정이라고 하니, 해당 부분을 기다려봐도 좋을 거 같습니다. 감사합니다.

**Reply by thebbong** [reply]
네. 기다리면 되는거같은데, 전에 vercel.app도메인에서는 잘 되다가 소유한도메인을 붙이니 이런 현상이 생겨 혹시 경험하신적 있나 해서 여쭤봤어요.
감사합니다. 😅

**Reply by hyesungoh** [reply]
도메인 주소를 변경하셨을 경우, 서치 콘솔의 주소 변경 기능을 이용하셨을까요?

저는 해당 경험이 없어 효과에 대해 말씀드리기는 어렵지만, 이런 기능도 있으니 한 번 봐보시면 도움이 되실 수도 있을 것 같아요!

![스크린샷 2022-08-03 오후 2 28 41](https://user-images.githubusercontent.com/26461307/182531058-1a618fce-09f0-4f36-8237-e6f6174a62bc.png)
